internal/lookout/repository/store.go

```diff
@@ -40,9 +40,10 @@ func (r *SQLJobStore) RecordJob(job *api.Job) error {
 }
 
 func (r *SQLJobStore) MarkCancelled(event *api.JobCancelledEvent) error {
-	_, err := upsert(r.db, "job",
-		"job_id", []string{"queue", "jobset", "cancelled"},
-		[]interface{}{event.JobId, event.Queue, event.JobSetId, event.Created})
+	_, err := r.db.Exec(`
```

Comment:
Is it intentional to drop the upsert operation here? Theoretically the events might be processed out of order and cancellation can be seen before the job is created - that's why the upsert was used.